2021-10-10-qubes-4-1-rc1.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "Qubes OS 4.1-rc1 has been released!"
+categories: releases
+download_url: /downloads/
+author: Andrew David Wong
+---
+
+After many years of work, the team is pleased to announce the first
+release candidate for Qubes 4.1!
+
+Qubes 4.1 includes several major new features, each of which is
+explained in depth in its own article:
+
+- [Qubes Architecture Next Steps: The GUI Domain]
+- [Qubes Architecture Next Steps: The New Qrexec Policy System]
+- [New Gentoo templates and maintenance infrastructure]
+- [Reproducible builds for Debian: a big step forward]
+
+This release candidate also includes numerous other improvements and
+bug fixes, which are listed in the [release notes] and in the [issue
+tracker].
+
+Finally, Qubes 4.1 features the following updated default components:
+
+- Xen 4.14
+- Fedora 32 in dom0
+- Fedora 34 template
+- Debian 11 template
+- Whonix 16 Gateway and Workstation templates
+- Linux kernel 5.10
+
+Qubes 4.1-rc1 is available on the [downloads] page.
+
+
+How to test Qubes 4.1-rc1
+-------------------------
+
+If you're willing to [test] this release candidate, you can help to
+improve the stable release by [reporting any bugs you encounter].
+Experienced users are strongly encouraged to join the [testing team]!
+
+
+Release candidate planning
+--------------------------
+
+As with any initial release candidate, it's likely that user testing
+will reveal important bugs that we'll want to fix before the stable
+release. Depending on the severity of the bugs discovered and how long
+it takes to fix them, we expect that it may be anywhere from a few weeks
+to a few months before we announce the second release candidate.
+
+On the other hand, we may be pleasantly surprised. If it turns out that
+no major issues are discovered in 4.1-rc1, we would then expect to
+announce the stable release of Qubes 4.1 in a few weeks.
```

Comment:
This is theoretically possible, but IMO highly unlikely. I'd remove this section, to not give a false hope (and re-add it with -rc2).